.github/workflows/build.yml

```diff
@@ -0,0 +1,36 @@
+name: build
+on: [push]
+jobs:
+  build:
+    runs-on: ubuntu-18.04
+    steps:
+      - name: Check Out Source Code
+        uses: actions/checkout@v2
+
+      - name: Set up Java 8
+        uses: actions/setup-java@v2
+        with:
+          distribution: 'temurin'
+          java-version: '8'
+          cache: 'maven'
+
+      - name: Install
+        run: |
+          echo 'Install on ' ${GITHUB_REF##*/}
+          ./mvnw install -DskipTests  -Dgpg.skip
```

Comment:
Why not run the unit tests here? Do we run them somewhere else?

Reply:
well, I did the same flow as travis has with only install, for example https://github.com/logzio/sawmill/blob/master/.travis.yml#L13

Reply:
That looks like a bug. Someone, somewhere, should be running the unit tests.

Reply:
I just checked some merged PR and it has only `install` and `deploy` stages https://travis-ci.org/github/logzio/sawmill/builds/753004033
If we want to have also test run I can add this stage if no objections.

Reply:
Let's finish migrating to GH actions as is. Once this works, let's enable tests as well.